LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017-2018 simplabs GmbH and contributors
+Copyright (c) 2017-2018 Mainmatter GmbH and contributors
```

Comment:
that seems incorrect tbh. in 2017/18 the company was still called "simplabs".

```suggestion
Copyright (c) 2017-2022 simplabs GmbH and contributors
Copyright (c) 2022-... Mainmatter GmbH and contributors
```

Reply:
> 2017-2022 Mainmatter GmbH

is fine – it's the same legal entity anyway just under a new name now.